client/js/verification-portal/index.js

```diff
@@ -19,9 +21,10 @@ function getFormComponent({ formType }) {
 	}
 }
 
-const VerificationPortal = ({ formType }) => {
+const ResourceView = ({ formType }) => {
```

Comment:
@maddiecousens I think it's good to break this out into a smaller component, but it's best practice to keep components in separate files.